### ISSUE TITLE
Split compat check in two parts

### ISSUE THREAD

**Issue by fredrikekre** [JuliaRegistries/RegistryCI.jl#73]
It would be useful to split the comment bullet on compat entries into one about dependencies with missing lower bound, and dependencies with missing upper bound.

**Comment by DilumAluthge** [comment]
Currently, we only check for upper bounds. We don't even look for lower bounds.

So we'd leave the current compat check unmodified. And we'd add a second compat check to look for lower bounds.

**Comment by fredrikekre** [comment]
Right, its not really possible to have missing lower bound. What I am really after is to differentiate between: (i) there is a (lower) bound but it is >= and (ii) there is no compat at all.
